<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glasssizeshow</class>
 <widget class="QWidget" name="glasssizeshow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 12pt;
color: blue;</string>
  </property>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>500</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/image/size.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_J">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>72</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_A">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>167</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_D">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>317</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_B">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>152</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_E">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>307</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_G">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>477</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_H">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>467</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_C">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>167</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_F">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>325</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_I">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>477</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>28</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>玻璃尺寸(mm)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
